src/main/java/ch/njol/skript/structures/StructAnnotate.java

```diff
@@ -0,0 +1,73 @@
+package ch.njol.skript.structures;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.MetaSyntaxElement;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.test.runner.TestMode;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.script.Annotation;
+import org.skriptlang.skript.lang.structure.Structure;
+
+@Name("Annotation (Structure)")
+@Description({
+	"A special metadata note visible to the next structure.",
+	"This does nothing by itself, but may change the behaviour of the following feature.",
+	"If the annotation does not exist (or the following feature does not use it) then it will have no effect.",
+	"There is no penalty for using annotations that do not exist."
+})
+@Examples({
+	"@my cool annotation",
+	"command /test:",
+	"",
+	"@my other cool annotation",
+	"on load:"
+})
+@Since("INSERT VERSION")
+public class StructAnnotate extends Structure implements MetaSyntaxElement {
+
+	static {
+		if (TestMode.ENABLED)
```

Comment:
Is this intentional? If so, this class should be moved to the test/runner package